docs/decisions/0003-adr-process.md

```diff
@@ -0,0 +1,109 @@
+# ADR Process
+
+- Status: Proposed
+- Deciders: Justice40 Team
+- Date: 2021-05-10
+- Tags: Process
+
+Technical Story: N/A
+
+## Context and Problem Statement
+
+We should document the process of creating an ADR to provide clarity to committers of this project.
+
+## Decision Drivers
+
+- **Clarity** - Identify historical reasoning behind decision-making
+- **Transparency** - Make decisions in the open
+- **Currency** - ADRs reflect the current state of the world, but can be superseded by later decisions. The highest-numbered ADR reflects the most recent decision around a given topic.
+- **Expediency** - We need good decisions, but good good software ships frequently. To balance these two factors, the ADR ensures we move forward in an informed way, but provides a way to self-correct as the world changes.
+
+## Considered Options
+
+- ADRs
+- Informal Conversations
+- Unilateral decision-making
+
+## Decision Outcome
+
+See Below for a way to use ADRs to make decisions on the project.
+
+### What is an ADR?
+
+An Architecture Decision Record is a text file, with a particular format, and an associated discussion. It is numbered for easy reference, and it has a general format.
+
+### When Should I Create an ADR?
+
+As outlined by Michael Nygard, ADRs represent "architecturally significant" decisions.
+
+The subject of an ADR should affect one of the following areas:
+
+- Structure
+- Dependencies
+- Interfaces
+- Construction techniques
+
+Small changes don't need one, but for larger changes that would introduce a new dependency or affect our overall system architecture, we recommend opening one.
+
+### Process
+
+Please see the below diagram:
+![ADR Process](./0003-files/adr_process-mmd.svg)
+
+1. **Start**: To create a new ADR, you have a few options:
+
+   1. **Old fashioned file copy**: Copy `template.md` into a new file in this directory, named `XXXX-name-of-issue.md` where the `XXXX` part is incremented from the previous ADR.
+
+   2. **ADR Tools**: Install `adr-tools` with `brew install adr-tools` then `adr new "Issue Name"` to make a new file. Copy contents of `template.md` in its place to get the right fields.
```

Comment:
I would add a bit about making sure you're in the right directory:

 Install `adr-tools` with `brew install adr-tools`. Then, in the `docs/decisions` directory, run `adr new "Issue Name"` to make a new file. Do not add anything to the opened markdown file yet; instead, copy the contents of `template.md` and paste into the new file to get the right fields.

Reply:
Actually, now that I've done this, I kind of feel like there isn't much benefit to using adr-tools since you still have to copy paste. You just get the right filename. Is there any other reason to use it?

Reply:
It's true, not using their default template takes a lot of the benefit away. I can remove for now.

High-level I'm beginning to wonder about capturing at least some of this using a project board that you can move issues around on and issue templates - we can discus this part further also